setup.cfg

```diff
@@ -0,0 +1,28 @@
+[metadata]
+name = libindic-ngram
+summary = N-Gram generator for Indic Languages
+description-file =
+  README.md
+author = Santhosh Thottingal
+author-email = santhosh.thottingal@gmail.com
```

Comment:
May be we can make this libindic developers?.. I know original code is written by @santhoshtr but there has been lot of changes contributed by many people. And we should some how acknowledge that.